src/FormElement/ItemSetSelect.php

```diff
@@ -44,4 +47,38 @@ public function applyToQuery(Query $query, array $data, array $formElementData):
             $query->addFacetFilter($formElementData['field_name'], array_filter($data['item_set_id']));
         }
     }
+
+    public function setApiManager(ApiManager $api): void
+    {
+        $this->api = $api;
+    }
+
+    public function getApiManager(): ApiManager
+    {
+        // var_dump(($this->api));
+        return $this->api;
+    }
+
+    public function stringifyData(array $data, array $formElementData) : string
+    {
+        $dataString = '';
+        $apiManager = $this->getApiManager();
+
+        if (!empty($data['item_set_id'])) {
+            $titles = [];
+            foreach ($data['item_set_id'] as $id) {
+                try {
+                    $itemSet = $apiManager->read('item_sets', $id)->getContent();
+                    $titles[] = $itemSet->title();
+                } catch (\Omeka\Api\Exception\NotFoundException $e) {
+                }
+            }
+
+            if (!empty($titles)) {
+                $description = "Item sets"; // @translate
+                $dataString = sprintf("%s : ( %s )", $description, implode(', ', $titles));
```

Comment:
I think `"Item sets"` won't be translated here as it is included in another string